README.md

```diff
@@ -490,6 +490,35 @@ If you specify both **alert** and **data**, target device will receive both noti
 * **520** `{"reason" : "unspecified"}` - the unknown error occurred.
 * **500** `{"reason" : reason}` - the server internal error occurred,
   specified by **reason**.
+ 
+### Healthcheck
+
+MongoosePush exposes a `/healthcheck` endpoint, from which you can get information about the current status of all connections in a `JSON` format, grouped by connection pool. An example with 2 pools, one being connected to the service and the other one not, would look like this:
+
+```json
+[
+{
+  "pool": "pool_name1",
+  "connection_status":
+    [
+      "connected",
+      "connected"
```

Comment:
So if I get 100 workers does it mean that this list is 100 "connected" strings?
How about:
```elixir
"connection_status" => %{
  "connected" => 2,
  "disconnected" => 0
}
```

Reply:
At first I thought we wanted to possibly give more information about the workers but since we stick with the connection status only I agree it's cleaner that way. However, the whole response is going to be restructured to match the draft RFC mentioned earlier